src/Ledger.Net/Requests/Concrete/Public Key Request/GetPublicKeyRequestBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Ledger.Net.Requests.Helpers;
+
+namespace Ledger.Net.Requests
+{
+    public abstract class GetPublicKeyRequestBase : RequestBase
+    {
```

Comment:
I can totally see why you'd change this. But, I'm keeping the Requests and Responses as clean so that they perfectly represent what is being sent to the device. I.e. the constructor will always take byte[] because that is what is sent to the device. The higher level methods like GetAddressAsync put a nice layer over the top so that they user doesn't need to worry about the raw data.